### ISSUE TITLE
flash.mk openocd script can't open target binary

### ISSUE THREAD

**Issue by Brandon-Hurst** [Analog-Devices-MSDK/msdk#872]
Attempting to flash my MAX78000FTHR via "make flash.openocd" as described in the [User Guide](https://analog-devices-msdk.github.io/msdk/USERGUIDE/#getting-started-with-command-line-development).  
![image](https://github.com/Analog-Devices-MSDK/msdk/assets/82959533/0dc6b004-cd36-4afb-89f4-93ab6bb1b75c)

Here is the output I consistently get:
![image](https://github.com/Analog-Devices-MSDK/msdk/assets/82959533/b00fe50e-5233-483f-8060-9b67c6cbf7d3)

I'm able to manually run openocd/arm-none-eabi-gdb server and flash the binary just fine, but the flashing script referenced here is unable to open the binary file. I also tried this from an Administrator-level bash environment.

**Comment by ethyljoybadiang** [comment]
Hello, I also have trouble similar to that. May I just ask, did you also encounter a gdb connection error?

**Comment by Brandon-Hurst** [comment]
Hi Ethyl,

No, I did not. I was able to flash my app over gdb-openocd. I ran it like it explains in the User Guide, below for your reference:

In one terminal window, run:
`openocd -s ${MAXIM_PATH}/Tools/OpenOCD/Scripts -f interface/cmsis-dap.cfg -f target/max78000.cfg`

In another terminal, cd into your project directory and run `arm-none-eabi-gdb` then run the following commands:
`target extended-remote :3333` connects to your running openocd client
`file build/max78000.elf` will set the target binary to load
`load` will load the flash

I was also able to flash with just OpenOCD by essentially running the same command as the flash.mk but on the command line. 

Incidentally, I also installed onto a VM for Ubuntu 22.04 yesterday and the flash.mk process works just fine on Linux. So it's perhaps a Windows issue.
